src/autolabel/configs/config.py

```diff
@@ -238,7 +239,16 @@ def label_selection(self) -> bool:
 
     def label_selection_count(self) -> int:
         """Returns the number of labels to select in LabelSelector"""
-        return self._prompt_config.get(self.LABEL_SELECTION_COUNT_KEY, 10)
+        k = self._prompt_config.get(self.LABEL_SELECTION_COUNT_KEY, 10)
+        if k < 1:
+            return len(self.labels_list())
+        return k
+
+    def label_selection_threshold(self) -> float:
+        """Returns the threshold for label selection in LabelSelector
+        If the similarity score ratio with the top Score is above this threshold,
+        the label is selected."""
+        return self._prompt_config.get(self.LABEL_SELECTION_THRESHOLD, 0.95)
```

Comment:
lets set this to be 0 by default? We should use the threshold only if provided.

Reply:
got it